app/main.py

```diff
@@ -1,3 +1,24 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    course_dict = {}
+
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+        self.course_dict["name"] = name
+        self.course_dict["description"] = description
+        self.course_dict["weeks"] = weeks
+
+    def days_to_weeks(days):
+        if days % 7 == 0:
```

Comment:
If you want to refactor your code, you may change it like this:
return ceil(days / 7)
Of course you have to import math before